backend/src/express.ts

```diff
@@ -0,0 +1,27 @@
+import express, {Request, Response } from 'express';
+import cookieParser from 'cookie-parser';
+import compression from 'compression';
+import morgan from 'morgan';
+import cors from 'cors';
+import apiRouter from './routes/api.router';
+// import healthRoute from './routes/health';
+
+const app = express();
+
+// Express middleware
+app.use(express.json());
+app.use(express.urlencoded({ extended: false }));
+// app.use(cookieParser());
+// app.use(compression());
+// app.use(morgan('dev'));
+app.use(cors());
+
+app.get('/api', (req: Request, res: Response) => {
+  res.send('Hello, TypeScript Express root');
+});
+
+// Routes
+app.use('/api', apiRouter);
+// app.use('/api/health', healthRoute);
```

Comment:
```suggestion
app.use('/api', apiRouter);
app.use('/api', [
  apiRouter
]);
```
You can use an array for this argument to base all your routers off the `/api` route that you defined. Using the health endpoint as an example, it will catch its `/health` part you defined, and become `/api/health` :) 
As you create more routers, add them to this array !

Reply:
I had the `/health` endpoint there in an earlier iteration and forgot to remove the commented out line. 
I'll keep this suggestion in mind for future similar cases.